### DISCUSSION TITLE
Viewing migrations inside collections tab

### DISCUSSION THREAD

**Discussion by sa902** [pocketbase/pocketbase#5716] [Q&A]
If I run migrations and seed in data it will store it correctly and I am able to run queries on it etc, so the table is there and the data is to. 
I am not able to find the created tables or data inside the collections tab. 

Am I doing something wrong or is this intentional? 
If I try to create a collection with the same table name it wont create it and shows an error message.

**Comment by ganigeorgiev** [comment]
While PocketBase uses SQLite under the hood you still need to follow it specifics data storage requirements.

Out of the box the Web API (and the Admin UI) knows only how to interact with [Collections](https://pocketbase.io/docs/collections/), aka. if you have imported your data as plain SQLite tables without creating the necessary Collections for them then it will not work.

See also https://pocketbase.io/docs/go-migrations/ and https://pocketbase.io/docs/go-collections/ (_note that with v0.23.0 there are breaking changes to the collection model and fields_).

**Reply by ganigeorgiev** [reply]
> If I try to create a collection with the same table name it wont create it and shows an error message.

That's because you've created the tables manually and not via the linked above collection APIs.

In any case, there is nothing really much I or anyone else that can assist you without more details about your migration and the steps you've taken to perform it.
